config/_shared.json

```diff
@@ -29,7 +29,7 @@
 	"mc_analytics_enabled": false,
 	"oauth_client_id": false,
 	"olark": {},
-	"port": false,
+	"port": 3000,
```

Comment:
Just leaving a note here. I've been tempted to make this line `process.env.PORT || 3000`. That's dangerous as the environment variable would _only_ override in the case that no env-specific port is defined.

This kind of change could, and perhaps should, happen in the config file. That would allow removing the conditions `env || config` that appear in several places.